frontend/src/scenes/onboarding/sdks/llm-analytics/manual.tsx

```diff
@@ -0,0 +1,217 @@
+import { useValues } from 'kea'
+import { useState } from 'react'
+
+import { LemonTabs } from '@posthog/lemon-ui'
+
+import { CodeSnippet, Language } from 'lib/components/CodeSnippet'
+import { apiHostOrigin } from 'lib/utils/apiHost'
+import { teamLogic } from 'scenes/teamLogic'
+
+import { SDKKey } from '~/types'
+
+import { DocumentationLink } from './DocumentationLink'
+
+export function LLMManualCaptureInstructions(): JSX.Element {
+    const { currentTeam } = useValues(teamLogic)
+    const [language, setLanguage] = useState<SDKKey>(SDKKey.API)
+
+    const languages = [
+        { key: SDKKey.API, label: 'API', lang: Language.Bash },
+        { key: SDKKey.NODE_JS, label: 'Node.js', lang: Language.JavaScript },
+        { key: SDKKey.PYTHON, label: 'Python', lang: Language.Python },
+        { key: SDKKey.GO, label: 'Go', lang: Language.Go },
+        { key: SDKKey.RUBY, label: 'Ruby', lang: Language.Ruby },
+        { key: SDKKey.PHP, label: 'PHP', lang: Language.PHP },
+    ]
+
+    return (
+        <>
+            <h3>Manual $ai_generation Event Capture</h3>
+            <p>For server-side SDKs, you can manually capture LLM events using the $ai_generation event.</p>
+
+            <LemonTabs
+                activeKey={language}
+                onChange={(key) => setLanguage(key as SDKKey)}
+                tabs={languages.map((l) => ({ key: l.key, label: l.label }))}
+            />
+
+            {language === SDKKey.NODE_JS && (
+                <>
+                    <h3>1. Install</h3>
+                    <CodeSnippet language={Language.Bash}>npm install posthog-node</CodeSnippet>
+
+                    <h3>2. Initialize PostHog</h3>
+                    <CodeSnippet language={Language.JavaScript}>{`import { PostHog } from 'posthog-node'
+
+const client = new PostHog('${currentTeam?.api_token}', {
+    host: '${apiHostOrigin()}'
+})`}</CodeSnippet>
+
+                    <h3>3. Capture $ai_generation Event</h3>
+                    <CodeSnippet language={Language.JavaScript}>{`// After your LLM call
+client.capture({
+    distinctId: 'user_123',
+    event: '$ai_generation',
+    properties: {
+        $ai_trace_id: 'trace_id_here',
+        $ai_model: 'gpt-4o-mini',
+        $ai_provider: 'openai',
+        $ai_input: [{ role: 'user', content: 'Your prompt' }],
```

Comment:
maybe can add same fun prompt about hedgehogs here - and maybe an actual fact in the response too :)

Reply:
Done!